browser/components/smartwindow/content/chat.mjs

```diff
@@ -541,8 +546,35 @@ class ChatBot extends MozLitElement {
           });
           continue;
         }
+        fullResponse += chunk;
+        // Extract title if this is the first message and we haven't captured it yet
```

Comment:
other parts of this file use `parseContentWithTokens` but this custom extraction is fine to land and we can refactor